mail_parent_recipient/tests/test_mail_parent_recipient.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Camptocamp
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.addons.mail.tests.common import TestMail
+
+
+class TestMailTemplate(TestMail):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestMailTemplate, cls).setUpClass()
+
+        cls.Users = cls.env['res.users']
+        cls.res_partner = cls.env['res.partner']
+        cls.mail_template = cls.env['mail.template']
+        cls.mail_comp_msg = cls.env['mail.compose.message']
+
+        # Company
+        company_vals = {
+            'name': 'company_name_test',
+            'email': 'company.mail.test@company'
+        }
+
+        cls.company_test = cls.res_partner.create(company_vals)
+
+        # Partners test 1 without email
+        partner_no_mail_vals = {
+            'name': 'partner_1',
+            'parent_id': cls.company_test.id,
+        }
+
+        # Partners test 2 with email
+        partner_with_mail_vals = {
+            'name': 'partner_2',
+            'email': 'partner.2.mail.test@company',
+            'parent_id': cls.company_test.id,
+        }
+
+        cls.partner_no_mail = cls.res_partner.create(partner_no_mail_vals)
+        cls.partner_with_mail = cls.res_partner.create(
+            partner_with_mail_vals
+        )
+
+    def create_mail_composer(cls, partner_to_send_id):
```

Comment:
```suggestion
    def create_mail_composer(self, partner_to_send_id):
```

In all the test methods too